src/Form/index.tsx

```diff
@@ -305,9 +305,11 @@ function Form({
 
   // All mount and unmount logic should live here
   useEffect(() => {
-    initState.renderCallbacks[_internalId] = () => {
-      setRender((render) => !render);
-    };
+    registerRenderCallback(_internalId, {
```

Comment:
can this function take three params - ID, key (form or loginProvider), and render callback?